docs/deployments/ASF-deployment-ci-cf.yml

```diff
@@ -0,0 +1,99 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Parameters:
+  CFBucket:
```

Comment:
I'd prefer something slightly more verbose, maybe `TemplateBucket`. Any thoughts on creating a template bucket as part of the stack, rather than taking it as an input?

Reply:
actually, `TemplateBucketName` might be even better, since I'm never sure whether parameters like this expect a Name or a full ARN.